src/app/(content-pages)/vpat/VPATPage.tsx

```diff
@@ -0,0 +1,2174 @@
+          </tr>
+
+          <tr>
+            <td className="border px-4 py-2 w-1/3">
+              <a
+                href="https://www.w3.org/TR/WCAG22/#redundant-entry"
+                className="underline"
+                target="_blank"
+                rel="noopener noreferrer"
+              >
+                <strong>3.3.7 Redundant Entry</strong>
+              </a>{" "}
+              (Level A 2.2 only)
+              <br />
+              Revised Section 508 – Does not apply
+            </td>
+            <td className="border px-4 py-2 w-1/3">Not applicable</td>
+            <td className="border px-4 py-2 w-1/3"></td>
+          </tr>
+
+          <tr>
+            <td className="border px-4 py-2 w-1/3">
+              <a
+                href="https://www.w3.org/TR/WCAG20/#ensure-compat-parses"
+                className="underline"
+                target="_blank"
+                rel="noopener noreferrer"
+              >
+                <strong>4.1.1 Parsing</strong>
+              </a>{" "}
+              (Level A)
+              <br />
+              Applies to:
+              <br />
+              WCAG 2.0 and 2.1 – Always answer ‘Supports’ WCAG 2.2 (obsolete and
+              removed) - Does not apply Revised Section 508
+              <ul className="list-disc ml-8">
+                <li>501 (Web)(Software)</li>
+                <li>504.2 (Authoring Tool)</li>
+                <li>
+                  <strong>602.3 (Support Docs)</strong>
+                </li>
+              </ul>
+            </td>
+            <td className="border px-4 py-2 w-1/3">Does not apply</td>
+            <td className="border px-4 py-2 w-1/3"></td>
+          </tr>
+
+          <tr>
+            <td className="border px-4 py-2 w-1/3">
+              <a
+                href="https://www.w3.org/TR/WCAG20/#ensure-compat-rsv"
+                className="underline"
+                target="_blank"
+                rel="noopener noreferrer"
+              >
+                <strong>4.1.2 Name, Role, Value</strong>
+              </a>{" "}
+              (Level A)
+              <br />
+              Also applies to:
+              <br />
+              Revised Section 508
+              <ul className="list-disc ml-8">
+                <li>501 (Web)(Software)</li>
+                <li>504.2 (Authoring Tool)</li>
+                <li>
+                  <strong>602.3 (Support Docs)</strong>
+                </li>
+              </ul>
+            </td>
+            <td className="border px-4 py-2 w-1/3">Partially supports</td>
+            <td className="border px-4 py-2 w-1/3">
+              Some elements found in the Find Properties page, such as the
+              filters, list box, toggle buttons & pagination does not announce
+              the change of state.
+            </td>
+          </tr>
+        </tbody>
+      </table>
+
+      <h3 className="heading-xl font-semibold my-2">
```

Comment:
I want to get your work merged so this isn't a blocker, but something we should either add in this PR or in a following one.

Right now when a screenreader user pulls up a list of all the tables on the page the tables don't have titles, so it looks like this:

![IMG_7862_2](https://github.com/CodeForPhilly/clean-and-green-philly/assets/7329799/0ddaed30-0f0c-49d0-83d1-d9ac9150d949)

If you add an id to this h3 tag, say `id="wcag-level-aa"`  and add an `aria-describedby="wcag-level-aa"` to the table tag on line 1032 then that will give the table a name and instead in the list it will look like:

![IMG_7863](https://github.com/CodeForPhilly/clean-and-green-philly/assets/7329799/146795c8-0f2d-4b10-8a6f-b075655bcb68)

Reply:
Hi @bacitracin, thank you for the feedback! I have linked titles to table descriptions in the latest commit. Can you pls re-review?

The titles are now read and displayed when I navigate to specific tables, as shown in the screenshot below. However, table titles are still not displayed on the menu itself like your screenshot shows. Do you think this would be an issue?

![table-titles](https://github.com/CodeForPhilly/clean-and-green-philly/assets/88219993/0b8e2433-8300-429d-a99a-666a385405f3)

Reply:
@zheyichn It is an issue because the shortcuts are how most screenreader users navigate the page, they usually don't read through line by line.

This is actually my bad, I meant `aria-labelledby` instead of `aria-describedby`. I was playing around with both and gave you the wrong one. Try that, and see if it will show up on the Voiceover tables page.

Reply:
@bacitracin Got it, that makes sense! I have replaced them with `aria-labelledby`, and the titles are displayed in the menu now.
![labelled-tables](https://github.com/CodeForPhilly/clean-and-green-philly/assets/88219993/f51d8a0e-5096-41ef-adbe-83db1df65ed8)